breeze

```diff
@@ -1977,6 +2003,16 @@ ${FORMATTED_INTEGRATIONS}
         init.sh. It will be executed after the environment is configured and started (in case
         --start-airflow is used).
 
+--skip-example-dags
+        Airflow example dags will not be included.
+
+--skip-default-connections
+        Airflow Default Connections will not be created.
+
+-T, --skip-tutorial-objects
+        Default Connections and sample dags will not be created. Use this to have a clean
+        Airflow without tutorial objects.
+
```

Comment:
Not sure but this section should be auto-generated by pre-commit

Reply:
No no  - this is fine here. This is the actual source of truth for all help documentation, so this is fine here.

Reply:
We do not have full sync betweeen the flags (in breeze-complete) and description of those (here), Maybe some day :)